using-valtimo/plugin/portaaltaak/configure-portaaltaak-plugin.md

```diff
@@ -0,0 +1,118 @@
+and change the status in the object to `ingediend`.
+4. The Objecten API should send a notification to the Notificaties API indicating the the object has been updated.
+5. Valtimo should be subscribed to notifications from the Notificaties API as part of the plugin setup and will receive
+the notification. When an update notification for a taak object is received Valtimo will check the status. If this is 
+set to `ingediend` the updated data from the taak object will be stored based on the value resolver configuration, and 
+the handling process will be started.
+6. As part of the handling process the [Complete Portaal taak](#complete-portaal-taak) action should be used. This 
+action will complete the task in Valtimo and update the zaak object status to `verwerkt`, indicating the the taak has 
+been processed. 
+   
+## Configure the plugin
+
+A plugin configuration is required before the plugin can be used. A general description on how to configure
+plugins can be found [here](../configure-plugin.md).
+
+If the Portaal taak plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the Portaal taak Plugin dependency can be found
+[here](/getting-started/modules/zgw/portaaltaak.md).
+
+To configure this plugin the following properties have to be entered:
+- **Notification API plugin.** Reference to another plugin configuration that will be used to receive notification on
+updates to objects. This is needed for completing tasks of which the status has been updated.
+- **Object management configuration.** Reference to the object management configuration that can be used to store the
+taak objects. If no option is available in this field an object management configuration has to be created first.
+- **Uploaded documents handler process.** Reference to the process that will be started to handle completion of tasks.
+This can do additional steps like handling the file attachments. A process task should be configured in this process 
+definition to handle the completion itself. This can be done using the [Complete Portaal taak](#complete-portaal-taak) 
+plugin action. The process 'Process Portaaltaak uploaded Documents' that is shipped with Valtimo can be used here. See 
+[this section](#configuring-the-task-completion-process ) on how to set up this process. 
+
+An example plugin configuration:
+![example plugin configuration](img/configure-plugin.png)
+
+## Configuring the task completion process 
+
+When a task object is updated and Valtimo gets the notification, a process is started to handle additional steps needed 
+for completing the task. Because the task can only be considered completed when all other processing of data has 
+happened, the actual completion of the task is also a plugin action that can be configured in this process.
+
+The process that is started in configured in the plugin properties by setting the 'Uploaded documents handler process'
```

Comment:
A bit confused as to what you're trying to say here.

Reply:
The grammar wasn't really correct. Please check again.